proto/storage/network_flow.proto

```diff
@@ -14,6 +14,9 @@ message NetworkFlow {
 
   // Need the clusterID as that is part of the key in RocksDB
   string cluster_id = 3; // @gotags: sql:"pk,type(uuid)"
+
+  // This field is set by Central everytime a flow is upserted.
+  google.protobuf.Timestamp updated_at = 4; // @gotags: sql:"index=brin"
```

Comment:
We can probably just use the default index here.

Reply:
I went with the brin index to be safe here. I'm not sure how expensive the pruning queries are but I'd bet they could get expensive in large clusters with a lot of flows.

Reply:
Do we need to ignore it somehow? e.g. `hash:"ignore" sensorhash:"ignore"` or do not expose in api?

Reply:
> Do we need to ignore it somehow? 
That's a good question. IMO we shouldn't use `storage` messages in APIs but I know that for this particular message we do (only in the internal API AFAIK). The thing is sensor should not be setting this field at all, so the value should be always `nil` when we are passing through the dedupers.